src/components/footer/index.jsx

```diff
@@ -1,21 +1,79 @@
 import Link from "next/link"
 import React from "react"
 
+
 export default function Footer() {
     return (
-        <footer className="sticky top-[100vh] p-4 h-[20vh] bg-white md:p-8 lg:p-10 dark:bg-gray-800">
-            <div className="mx-auto max-w-screen-xl text-center">
-                <p className="my-6 text-gray-500 dark:text-gray-400">
+        <footer class="text-center bg-gray-800 text-white">
```

Comment:
the footer must have a light and dark version, as before.

It also needs to get back to being responsive.

Other than that, it was a great job. I put some reference links, take a look if possible

Reply:
I've made another commit with changes to the file and also checked the responsiveness from the dev tools (responsiveness isnt my strong suit haha.) The light and dark versions have also been added. I hope it is good to go now?

Reply:
amazing!